css/components/alert.css

```diff
@@ -0,0 +1,84 @@
+@import "../base/all.css";
+
+.dflex{
+    display: flex;
+    align-items: center;
+}
+
+.flex-y{
+    flex-direction: column;
+}
+
+.flex-ai-start{
+    align-items: flex-start;
+}
+
+.m-left-auto{
+    margin-left: auto;
+}
+
+.alert{
+    padding:16px;
+    margin:16px;
+    border:1px solid transparent;
+    word-wrap: break-word;
+}
+
+.alert-info-icon{
+    margin-right: 16px;
+    font-size: 24px;
+}
+
+.alert-close-icon{
+    cursor: pointer;
+    border: 0;
+    background-color: transparent;
+    outline: 0;
+    color:rgba(0, 0, 0, 0.45);
+    font-size: 12px;
+}
+
+.alert-close-icon:hover{
+    color:#000000bf;
```

Comment:
use css variables for color -> var(--color-hover) as you have made the css variables, so that must be used

Reply:
This got missed, I will add it as css-variable.